### ISSUE TITLE
Invalid links to media when placed in Modular Pages

### ISSUE THREAD

**Issue by pikooh** [getgrav/grav#3278]
Hi!

I'm trying to link to an uploaded PDF file in a module (I want a long modular page with all "pages" in one) - I followed the [instructions](https://learn.getgrav.org/17/content/modular) pretty close, I think :) The website itself works otherwise.

Steps I did:
* create a page (in the root folder) and module (in another page) and saved them
* uploaded an image and a PDF (`mars.png` and `ausser_kontrolle.pdf`) in both
* clicked the `+` symbol next to the files and added them to both the page and the modular page
* saved the pages again and checked them in the frontend

What I expected:
* Both the page and the modular page's section contain valid links to the image and the PDF

What happened:
* Only the page contains the correct links to the image and the PDF. The links in the module are broken.

Screenshots:
Left: Added the files in a page; Right: the files are linked correctly when accessing the page
![image](https://user-images.githubusercontent.com/1488366/111539789-93c14000-876e-11eb-942f-49caf6d1b612.png)

Left: Added the files in a module; Right: the files are linked incorrectly when accessing the modular page
![image](https://user-images.githubusercontent.com/1488366/111540053-e995e800-876e-11eb-8eef-ba18dfa3a2c2.png)
(the displayed image is from the default text template, which displays the first image next/below the content - this works, funnily enough)

I'm using the default Quark Theme 2.0.3 on Grav 1.7.7.

Folder structure when I access the server with FTP:
![image](https://user-images.githubusercontent.com/1488366/111540465-793b9680-876f-11eb-9730-352073c71295.png)

Is there anything I'm doing wrong?

**Comment by mahagr** [comment]
I'm not able to reproduce the issue with Quark theme using the latest version of Grav.

**Comment by pikooh** [comment]
Thanks for checking it out! Any idea what else I could do to either make it reproducible or fix it?

One thing I didn't mention in the bug description: I'm using the numeric prefix ordering for the modular pages, but also without that it doesn't fix the link.
![image](https://user-images.githubusercontent.com/1488366/111919814-283be300-8a8c-11eb-99e9-4b6c4aa45973.png)

**Comment by mahagr** [comment]
Yeah, I was using modular home page, which had a numeric prefix in it, meaning that the path doesn't match the route.
